rest/handler.go

```diff
@@ -734,6 +734,7 @@ func (h *handler) writeStatus(status int, message string) {
 	if status < 300 {
 		h.response.WriteHeader(status)
 		h.setStatus(status, message)
+		h.setHeader("Content-Type", "application/json")
```

Comment:
`h.writeStatus` isn't writing any response body for a successful status code, so I'm not sure we should be setting the content-type header here.

There are instances where `writeStatus` is called with an intentionally plain text response (e.g. `handleGetRevTree()`)

Reply:
Makes sense. Fixed in latest revision.